### PR TITLE
Button change effect app

### PR DESCRIPTION
**Pull request by RiyaGupta89**
# Resolve issue #144 
Added the buttons color changing effect project in the repository. 
*This web app allows a user to select any one color from the options that are being to the user and as the user clicks on the color all the button changes to that particular color.*
challenge1
challenge2

### CODE REVIEW

**Comment by harshita2216** [comment]
@RiyaGupta89, please update the description and mention the issue number. Thanks.

**Comment by RiyaGupta89** [comment]
@harshita2216  Done. I edited my message. Please have a look.

**Review by harshita2216 — CHANGES_REQUESTED** [review]
Please update index.md and add a readme. @RiyaGupta89

**Comment by RiyaGupta89** [comment]
@harshita2216 Where is index.md?

**Comment by RiyaGupta89** [comment]
dir
Can't find index.md, which command should i run?

**Comment by Ayushparikh-code** [comment]
> dir
> Can't find index.md, which command should i run?

@RiyaGupta89  do again fetch the upstream of original repo & merge it to ur cloned repo then check again hope u will find that folder ...

**Comment by RiyaGupta89** [comment]
@Ayushparikh-code  @harshita2216  Added the project in index.md also added a readme file. Please review my PR.

**Comment by RiyaGupta89** [comment]
How can I resolve the issue? Please help.

File: JAVASCRIPT BUTTON APP/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- Have a look at the webapp.
+
+
+##Screenshots
+
+![Demo1](public/images/challenge1.png)
+
+<br>
+
+![Demo2](public/images/challenge2.png)
```
Comment (harshita2216) [inline JAVASCRIPT BUTTON APP/README.md:41]: Image doesn't show. Please check?

Comment (harshita2216) [reply JAVASCRIPT BUTTON APP/README.md:41]: Also the screenshots doesn't follow markdown. Check the preview please?

**Comment by harshita2216** [comment]
@RiyaGupta89, update your fork please. Then add your project in the index.md file.

**Comment by harshita2216** [comment]
@RiyaGupta89, I have resolved the conflicts. Please check the image only then we'll merge it.

**Comment by RiyaGupta89** [comment]
image
I can see my images here, i don't know why its not visible there. I did the same in survey form website,  there it worked completely fine.

**Comment by RiyaGupta89** [comment]
Or else merge this one, i will create a seperate issue for correcting images, if there are any problem with images.

**Comment by RiyaGupta89** [comment]
@harshita2216 Oh, okay. Thanks alot.

**Review by harshita2216 — APPROVED** [review]

**Pull request merged**